ansible/roles/test/files/acstests/lag_test.py

```diff
@@ -214,6 +214,10 @@ def run_ptf_to_dut_traffic_test(self):
             src_ip = self.ptf_lag['ip'].split('/')[0]
             dst_ip = self.dut_vlan['ip'].split('/')[0]
 
+            arp_pkt = Ether(src= src_mac, dst=dst_mac)/ARP(op=ARP.who_has, psrc=src_ip, pdst=dst_ip, hwsrc=src_mac)
+            send(self, port_behind_lag, arp_pkt)
+            time.sleep(1)
```

Comment:
Commonly, in sonic test scripts, the "arp responder" is used to give correct ARP response to DUT.
Here, seems trying to use a different way by sending an ARP reqeust or Gratuitous ARP.  Maybe that's not a good solution since it will reduce the step of the ARP request from DUT and it may be not a normal behavior in networking.     If the "arp request" from dut is also one necessary step for this whole testcase, we should not bypass it.